bundles/org.openhab.binding.echonetlite/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,378 @@
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="standardVersionInformation">
+		<item-type>String</item-type>
+		<label>Standard Version Information</label>
+		<description>Standard Version Information</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="identificationNumber">
+		<item-type>String</item-type>
+		<label>Identification Number</label>
+		<description>Identification Number</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="manufacturerFaultCode">
+		<item-type>String</item-type>
+		<label>Manufacturer Fault Code</label>
+		<description>Manufacturer Fault Code</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="faultStatus">
+		<item-type>Switch</item-type>
+		<label>Fault Status</label>
+		<description>Fault Status</description>
+		<category>Alarm</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="faultDescription">
+		<item-type>String</item-type>
+		<label>Fault Description</label>
+		<description>Fault Description</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="manufacturerCode">
+		<item-type>String</item-type>
+		<label>Manufacturer Code</label>
+		<description>Manufacturer Code</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="businessFacilityCode">
+		<item-type>String</item-type>
+		<label>Business Facility Code</label>
+		<description>Business Facility Code</description>
+		<category>Text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="powerSavingOperationSetting">
+		<item-type>Switch</item-type>
+		<label>Power Saving Operation Setting</label>
+		<description>Power Saving Operation Setting</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<channel-type id="cumulativeOperatingTime">
+		<item-type>Number</item-type>
+		<label>Cumulative Operating Time</label>
+		<description>Cumulative Operating Time</description>
+		<category>Number</category>
+	</channel-type>
+
+	<channel-type id="airFlowRate">
+		<item-type>String</item-type>
+		<label>Air Flow Rate</label>
+		<description>Air Flow Rate</description>
+		<category>Flow</category>
+		<state>
+			<options>
+				<option value="Auto">Auto</option>
+				<option value="Rate 1">Rate 1</option>
+				<option value="Rate 2">Rate 2</option>
+				<option value="Rate 3">Rate 3</option>
+				<option value="Rate 4">Rate 4</option>
+				<option value="Rate 5">Rate 5</option>
+				<option value="Rate 6">Rate 6</option>
+				<option value="Rate 7">Rate 7</option>
+				<option value="Rate 8">Rate 8</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="automaticControlOfAirFlowDirection">
+		<item-type>String</item-type>
+		<label>Automatic Control Of Air Flow Direction</label>
```

Comment:
Please try to shorten this label (< 25 characters, 2-3 words). Description will still explain the details. Perhaps "Air Flow Direction"?

Reply:
I've reduced it to "Automatic Air Flow Direction".  Unfortunately "Air Flow Direction" is not correct in this case as the channel controls whether the direction control is automated, rather that what the air flow direction is (which are a separate fields).

Reply:
TBH, I'm not sure if this is a rule of thumb rather than a hard limit. Let's close comment and hope it will fit in all UI's.